bijection-core/src/main/scala/com/twitter/bijection/StringInjections.scala

```diff
@@ -53,7 +53,7 @@ trait StringInjections extends NumericInjections {
           val decBuf = decRef.get
           val dec = decBuf._1
           val buf = decBuf._2
-          val maxSpace = (b.length * dec.maxCharsPerByte).toInt + 1
+          val maxSpace = (b.length * (dec.maxCharsPerByte.toDouble)).toInt + 1
```

Comment:
actually, the bug is that dec.maxCharsPerByte returns a `Float`. That can only store all integers up to `2^{24}`. After that it steps up by 2. So the toInt truncates by 2 in that case, not rounding down by one.

I don't know what `2B` means in this case. Can you explain?